app/src/pages/dashboard/index.tsx

```diff
@@ -88,6 +88,21 @@ function DashboardBase({ className }: DashboardProps) {
   const { data: { moneyMarketEpochState } = {} } = useEarnEpochStatesQuery();
   const { data: marketUST } = useMarketUstQuery();
   const { data: marketANC } = useMarketAncQuery();
+  const ancPriceRelevantHistory = useMemo(() => {
```

Comment:
I just noticed that the API itself only returns the last 12 months of data anyway, so we dont really need to do this. However, we can keep it in.

The is an edge case here that could make things go awry as you are assuming that the history is sorted in chronological order for this to work.

I think

```
return history.filter(point => point >= yearAgoTimestamp) is safer that looking for a point and slicing.
```

Reply:
We should probably also sort so we dont make assumptions of the API;

```
return history.filter(point => point >= yearAgoTimestamp).sort((a, b) => a.timestamp - b.timestamp)
```

Reply:
✅ https://github.com/Anchor-Protocol/anchor-web-app/pull/308/commits/56ad3f4fa2aecdc386d286891ff3d9141d847d1f